### ISSUE TITLE
Proof points should include measures of the extent to which processes are successful

### ISSUE THREAD

**Issue by jasonjgw** [w3c/maturity-model#91]
The proof points should include measures of the extent to which the processes pursued by the organization lead to successful outcomes over time with respect to accessibility.

In the ICT development life cycle section, for instance, measures should be taken of the extent to which what is actually produced satisfies the technical accessibility standards that have been adopted by the organization. An additional, relevant measure is concerned with the timeliness and correctness of changes made to resolve accessibility-related defects found during testing or after deployment.

In the procurement section, there should be a proof point that measures whether the technological products actually acquired consistently satisfy the accessibility standards adopted by the organization. Improvement in this metric over time would furnish evidence that the procurement procedures and requirements are effective.

User research is discussed (section 3.4.1.3), but the outcomes of the user research aren't. Does the research indicate high levels of accessibility? Are accessibility-related issues identified via user research actually addressed in a timely and effective manner?

The only reference to the degree to which technical standards are met that I could find is in the "Accessible direct Communication" section (i.e., section 3.1.1.2 of the current draft). There does not appear to be any proof point related to the organization's handling of defects; yet, the degree to which accessibility-related defects are resolved in a timely and effective manner is a central distinction between organizations at different levels of maturity in this domain.

I am concerned that an organization could satisfy many of the proof points without consistently succeeding in its accessibility efforts - and without consistently improving its performance over time. None of the other indicators matters if the organization's success in acquiring and delivering accessible technology-based products and services remains consistently low, yet this is not measured in most of the contexts to which it is relevant. Much of the "evidence" of maturity described in the document is circumstantial, even in those cases in which more direct evidence exists (e.g., improvements in technical standards conformance over time, or the effective and resolution of accessibility-related defects).

**Comment by Helixopp** [comment]
We have added the following Proof Point to every Dimension: Dimension-related goals are established, metrics are defined, and progress is tracked. We do not tell people what those metrics or goals should be, or how how to track them but require them to do so. We believe this addresses your issue. If you disagree please reopen
